compiler/ilgen/TypeDictionary.hpp

```diff
@@ -36,9 +36,9 @@
 #include "ilgen/IlBuilder.hpp"
 
 class TR_Memory;
-class TR_HashTabString;
 
 namespace OMR { class StructType; }
+namespace OMR { class UnionType; }
```

Comment:
Yeah, I wasn't too happy either when I added them. Technically, we _could_ still avoid this declaration by using Pimpl pattern. But I doubt if it would be worthwhile doing it considering the tradeoffs.